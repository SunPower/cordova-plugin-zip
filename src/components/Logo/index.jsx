import React from 'react'
import SunPowerImage from '../SunPowerImage'
import './Logo.scss'

function Logo() {
  return (
    <div className="Logo container">
      <div className="columns is-centered">
        <div className="column is-half">
          <SunPowerImage />
        </div>
      </div>
    </div>
  )
}

export default Logo
