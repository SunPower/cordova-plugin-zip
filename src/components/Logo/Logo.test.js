import React from 'react'
import { shallow } from 'enzyme'
import Logo from '.'

describe('Logo component', () => {
  it('renders without crashing', () => {
    const component = shallow(<Logo />)
    expect(component).toMatchSnapshot()
  })
})
