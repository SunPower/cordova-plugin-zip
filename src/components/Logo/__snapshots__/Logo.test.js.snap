// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component renders without crashing 1`] = `
<div
  className="Logo container"
>
  <div
    className="columns is-centered"
  >
    <div
      className="column is-half"
    >
      <SunPowerImage />
    </div>
  </div>
</div>
`;
